src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Value/NullableConverter.cs

```diff
@@ -1,73 +1,102 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.Diagnostics;
-
 namespace System.Text.Json.Serialization.Converters
 {
     internal sealed class NullableConverter<T> : JsonConverter<T?> where T : struct
     {
+        internal override ConverterStrategy ConverterStrategy { get; }
```

Comment:
```suggestion
        internal override ConverterStrategy ConverterStrategy => _elementConverter.ConverterStrategy;
```

Reply:
Using an auto property is intentional here. The particular property is invoked by the base constructor before `_elementConverter` can be populated.

Reply:
What ConverterStrategy is the nested value type held by Nullable? Is it always "Value" (which implies read-ahead on not resumable)?